src/nunit-gui/Views/MainForm.cs

```diff
@@ -163,5 +163,11 @@ protected override void OnDragDrop(DragEventArgs drgevent)
             if (files != null)
                 FilesDragAndDropped?.Invoke(files);
         }
+
+        protected override void OnDragEnter(DragEventArgs drgevent)
+        {
+            if (drgevent.Data.GetDataPresent(DataFormats.FileDrop))
+                drgevent.Effect = DragDropEffects.Copy;
+        }
```

Comment:
This should be part of drag and drop I guess.

I'm seeing three different things in this commit. I'd like to leave out one. I can do it but it's more work than if you kept things isolated.